File: backend/src/main/java/com/back/ourlog/domain/statistics/dto/EmotionGraphResponse.java
package com.back.ourlog.domain.statistics.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;

import java.util.List;

@Getter
@AllArgsConstructor
public class EmotionGraphResponse {
    private List<EmotionLineGraphDto> emotionLineGraph; // 선 차트용 데이터
    private List<EmotionRankDto> emotionRanking;  // 순위 차트용 데이터
}

File: backend/src/main/java/com/back/ourlog/domain/statistics/dto/EmotionGraphResponse.kt
package com.back.ourlog.domain.statistics.dto

data class EmotionGraphResponse(
    val emotionLineGraph: List<EmotionLineGraphDto>, // 선 차트용 데이터
    val emotionRanking: List<EmotionRankDto>         // 순위 차트용 데이터
)